src/_actions/assetType.action.js

```diff
@@ -5,7 +5,7 @@ const { LOAD_ASSET_TYPE_SUCCESS, LOAD_ASSET_TYPE_FAILURE } = constants;
 
 export const loadAssetTypeAction = (page, limit = 10) => {
   return (dispatch) => {
-    axios.get(`assetTypes?_page=${page}&_limit=${limit}`)
+    axios.get('asset-types/')
```

Comment:
Just to clarify, I have been told that all URLs should be in an env file. Do you know about this? Please ask Joanne for more details

Reply:
The base url is in the env file. tho only thing we specify are the resource urls.

Reply:
Make sense. Thanks for clarifying